Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -30,11 +38,76 @@ public static class TextTable
         /// <summary>
         /// Build a table for given message with given padding.
         /// Padding means how many spaces will a message be wrapped with.
-        /// Table itself is made of: "+-" symbols. 
+        /// Table itself is made of: "+-" symbols.
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (string.IsNullOrEmpty(message))
+            {
+                return "";
+            }
+
+            var longestWordInMessageSize = LongestWordInMessageSize(message);
+
+            var sb = new StringBuilder();
+
+            // create a padded line
```

Comment:
Could be a function. Also, it could simply be a string.

Remember, ever new string builder is an overhead. Don't abuse it. Try to re-use it. Remember the benchmark of string lesson? Only when you append string (+=) 10 times or more, it start being worth using StringBuilder.